sdcm/cluster.py

```diff
@@ -3339,6 +3340,8 @@ def remove_sct_metrics(x):
             scrape_configs = templ_yaml["scrape_configs"]
             scrape_configs.append(dict(job_name="stress_metrics", honor_labels=True,
                                        static_configs=[dict(targets=loader_targets_list)]))
+            scrape_configs.append(dict(job_name="gemini_metrics", honor_labels=True,
```

Comment:
run this only if Gemini is intended to run

Reply:
fixed